### DISCUSSION TITLE
The AWS Access Key Id you provided does not exist in our records.

### DISCUSSION THREAD

**Discussion by Lunana0110** [nextflow-io/nextflow#4418] [General]
Hi,

I'm trying to read data from S3 bucket, but it errors with "The AWS Access Key Id you provided does not exist in our records."

I could sync and cp data from s3(exactly same files) in my terminal, any thoughts on this? 

Here is the log files:
[error.log](https://github.com/nextflow-io/nextflow/files/12921284/error.log)

Thanks

**Comment by bentsherman** [comment]
Try using Nextflow 23.10, I think some issues with AWS authentication were fixed since 23.04

**Comment by t-whalley** [comment]
I am having a similar problem where if I run nextflow directly from the terminal it runs, but if I call it as part of a bash script I have the same error. I am using nextflow 23.10.1.5891.

**Reply by bentsherman** [reply]
This error usually means some part of your credentials are missing. Do you also have a session token in addition to access key and secret key?

**Comment by AsanchezMD** [comment]
Hello, I am having this issue as well, any thoughts on how to resolve this?

**Reply by ICE0584** [reply]
me too and i do not know how to solve it? did anyone fix it?
